.gitignore

```diff
@@ -106,4 +106,6 @@ ENV/
 site/
 Pipfile
 Pipfile.lock
-.icloud
\ No newline at end of file
+.icloud
+
+TODO
```

Comment:
These should probably be in your [global .gitignore](https://stackoverflow.com/a/7335487/1003288)

Reply:
You are right and didn't knew about global .gitignore haha thanks for pointing it out 😄